metadata-ingestion/src/datahub/ingestion/source/source_registry.py

```diff
@@ -92,3 +92,9 @@
     source_registry.register("mongodb", MongoDBSource)
 except ImportError as e:
     source_registry.register_disabled("mongodb", e)
+
+try:
+    from .glue import GlueSource
+    source_registry.register("glue", GlueSource)
+except ImportError as e:
+    source_registry.register_disabled("glue", e)
```

Comment:
The ingestion framework now discovers plugins through entries in the package metadata (#2316), so you can just add a new entry in `setup.py` (after you rebase):

```python
"datahub.ingestion.source.plugins": [
    ...
    "glue = datahub.ingestion.source.glue:GlueSource",
    ...
],
```